doc/asciidoc/pagerank.adoc

```diff
@@ -118,97 +120,100 @@ As we might expect, the Home page has the highest PageRank because it has incomi
 ----
 
-.parameters
+.Parameters
 [opts="header",cols="1,1,1,1,4"]
 |===
-| name | type | default | optional | description
-| label  | string | null | yes | label to load from the graph. If null load all nodes
-| relationship | string | null | yes | relationship-type to load from the graph. If null load all relationships
-| iterations | int | 20 | yes | how many iterations of page-rank to run
-| concurrency | int | available CPUs | yes | number of concurrent threads
-| dampingFactor | float | 0.85 | yes | damping factor of the page-rank calculation
-| write | boolean | true | yes | if result should be written back as node property
-| writeProperty | string | 'pagerank' | yes | property name written back to
-| graph | string | 'heavy' | yes | use 'heavy' when describing the subset of the graph with label and relationship-type parameter, 'cypher' for describing the subset with cypher node-statement and relationship-statement
+| Name | Type | Default | Optional | Description
+| label  | string | null | yes | The label to load from the graph. If null, load all nodes
+| relationship | string | null | yes | The relationship-type to load from the graph. If null, load all relationships
+| iterations | int | 20 | yes | How many iterations of PageRank to run
+| concurrency | int | available CPUs | yes | The number of concurrent threads
+| dampingFactor | float | 0.85 | yes | The damping factor of the PageRank calculation
+| write | boolean | true | yes | Specify if the result should be written back as a node property
+| writeProperty | string | `pagerank` | yes | The property name written back to
+| graph | string | `heavy` | yes | Use `heavy` when describing the subset of the graph with label and relationship-type parameter. Use `cypher` for describing the subset with cypher node-statement and relationship-statement
 |===
 
-.results
+.Results
 [opts="header",cols="1,1,6"]
 |===
-| name | type | description
-| nodes | int | number of nodes considered
-| iterations | int | number of iterations run
-| dampingFactor | float | damping factor used
-| writeProperty | string | property name written back to
-| write | boolean | if result was written back as node property
-| loadMillis | int | milliseconds for loading data
-| computeMillis | int | milliseconds for running the algorithm
-| writeMillis | int | milliseconds for writing result data back
+| Name | Type | Description
+| nodes | int | The number of nodes considered
+| iterations | int | The number of iterations run
+| dampingFactor | float | The damping factor used
+| writeProperty | string | The property name written back to
+| write | boolean | Specify if result was written back as node property
+| loadMillis | int | Milliseconds for loading data
+| computeMillis | int | Milliseconds for running the algorithm
+| writeMillis | int | Milliseconds for writing result data back
 
 |===
 
 
-.running algorithm and streaming results
+Use the following to run the PageRank algorithm, and stream results:
```

Comment:
Does it help the reader to repeat the term "PageRank" time and again while they are in the PageRank section?